packages/rocket-batch-file-process-aws-infrastructure/README.md

```diff
@@ -7,11 +7,17 @@ Every time a new File is uploaded to the Source S3 Bucket, a new Lambda function
 This Lambda function will
 
 - Split the source file in smaller chunks. The chunk size is defined by the user as input parameter of the rocket.
-- Persist the new formed chunk in the Staging Bucket.
+- Persist the new formed chunk file in the Staging Bucket.
 - Persist a new event (containing fileUri and fileSize) for each chunk in the Event Store.
 
-**Disclaimer:** As of now the rocket can only process CSV files.
+Then, for every chunk file dropped in the Staging rocket a new Lambda function will be triggered that will:
 
+- Read each chunk file line by line and persist a new event in the Events Store for each row
+
+This event is defined in the Booster application and can be consumed from the Event Handler for any kind of processing.
```

Comment:
I'd rephrase this a little bit for consistency:

```suggestion
This event is registered in the Booster application's event store as a regular event named `RowAdded`. Then, you can create event handlers to perform any kind of processing.
```

I'd also put an example of a full event handler processing a `RowAdded` event here.

Reply:
Yes totally! I will re-write the whole doc with a full end to end example. It is not clear now.